Description: Avoid configure script special-casing 64bit arch library path
Last-Update: 2010-03-10
Index: jack-audio-connection-kit/configure.ac
===================================================================
--- jack-audio-connection-kit.orig/configure.ac
+++ jack-audio-connection-kit/configure.ac
@@ -171,19 +171,8 @@
 
 #
 # We need to establish suitable defaults for a 64-bit OS
+# But we don't use /usr/lib64 in Debian
 libnn=lib
-case "${host_os}" in
-    linux*)
-    case "${host_cpu}" in
-	x86_64|mips64|ppc64|sparc64|s390x)
-        libnn=lib64
-	;;
-    esac
-    ;;
-    solaris*)
-    ## libnn=lib/sparcv9 ## on 64-bit only, but that's compiler-specific
-    ;;  
-esac
 
 ## take care not to  override the command-line setting
 if test "${libdir}" = '${exec_prefix}/lib'; then
